<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign</class>
 <widget class="QDialog" name="sign">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#sign{border-image: url(:/zxc/image/8211.jpg);}</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>341</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>330</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(0, 255, 255);</string>
   </property>
   <property name="placeholderText">
    <string>请输入用户名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>330</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/zxc/image/2-21.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>380</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid rgb(0, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
   <property name="placeholderText">
    <string>请输入新密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>380</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/zxc/image/2-1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170, 85, 255);
font-size:20px;
border:1px solid rgb(230, 234, 255);
background-color:rgb(220, 219, 211);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170, 85, 255);
font-size:20px;
border:1px solid rgb(230, 234, 255);
background-color:rgb(220, 219, 211);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>430</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170, 85, 255);
font-size:20px;
border:1px solid rgb(230, 234, 255);
background-color:rgb(220, 219, 211);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>380</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170, 85, 255);
font-size:20px;
border:1px solid rgb(230, 234, 255);
background-color:rgb(220, 219, 211);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>330</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(170, 85, 255);
font-size:20px;
border:1px solid rgb(230, 234, 255);
background-color:rgb(220, 219, 211);
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>更新</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
